portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,10 +59,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Retreives textarea values.
+    // Retreives the entry values.
     String comment = request.getParameter("textarea");
+    String firstName = request.getParameter("firstName");
+    String lastName = request.getParameter("lastName");
+    long timestamp = System.currentTimeMillis();
     // TODO(aabundis): Verify that a comment was written, no empty submissions.
-    comments.add(comment);
+
+    // Creates an Entity for each comment entry.
+    Entity entryEntity = new Entity("entry");
+    entryEntity.setProperty("firstN",firstName);
+    entryEntity.setProperty("lastN",lastName);
+    entryEntity.setProperty("com",comment);
```

Comment:
Consider using expanded property names (ie. firstName, lastName, comment) - it increases readability!

Reply:
Done!